sinabs/backend/dynapcnn/chips/speck2e.py

```diff
@@ -28,8 +29,3 @@ def get_output_buffer(cls):
     @classmethod
     def set_kill_bits(cls, layer: DynapcnnLayer, config_dict: dict) -> dict:
         return config_dict
-
-    @classmethod
-    def get_dynapcnn_layer_config_dict(cls, layer: DynapcnnLayer):
```

Comment:
why speck2e doesnt have get_dynapcnn_layer_config anymore?

Reply:
If I'm not missing anything, this method just inherited its parent without changing any of the behavior, so we can just omit it as well. It should not change any behavior.

Reply:
I just double-checked that all the other classes are not changing the parent's behavior class.

To be precise: 
The other classes add a:
```
config_dict.pop("weights_kill_bit")
config_dict.pop("biases_kill_bit")
config_dict.pop("neurons_value_kill_bit")
```

However, in the parent class, we find:
```
# config_dict["weights_kill_bit"] = torch.zeros_like(weights).bool().tolist()
# config_dict["biases_kill_bit"] = torch.zeros_like(biases).bool().tolist()
# "neurons_value_kill_bit" : torch.zeros_like(neurons_state).bool().tolist()
```

these entries will only be generated/updated when calling `set_kill_bits`.